### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#175] [Esercizi - Viaggi/Piazza] [answered]
![Es 3](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/3ba73804-7b33-4499-aee3-ac1afe182954)

**Comment by Elia-Belli** [comment]
Un gruppo deve soddisfare 3 proprietà:
- **Associatività:** siano $f,g,h \in G$, $f \circ (g \circ h) = (f \circ g) \circ h$ (proprietà della composizione);
- **Elemento neutro**: la funzione identità $Id_A$ è l'elemento neutro per la composizione: $Id_X \circ f = f = f \circ Id_X$;
- **Esistenza dell'inverso**: siccome $f \in G$ è una bigezione allora esiste sempre $f^{-1}| f \circ f^{-1} = Id_A= f^{-1} \circ f$

**Comment by CiottoloMaggico** [comment, marked as answer]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/e40c5eac-2756-4409-9712-9ec4bc4a2d0b)

**Comment by daromacarlo** [comment]
![Immagine 2023-12-05 194546](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/0bfa4cfb-1aaa-438d-a949-8eacd1d8878a)
![Immagine 2023-12-05 194600](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/635fb9d4-da1d-4abd-ae56-e072e08aabdb)
![Immagine 2023-12-05 194613](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/7c99b510-6028-482d-bcfb-0d128c6842c8)
